main.go

```diff
@@ -481,6 +491,73 @@ func parseBoolFilter(value string, boolFilter *tracee.BoolFilter) error {
 	return nil
 }
 
+func parseArgFilter(filterName string, operatorAndValues string, eventsNameToID map[string]int32, argFilter *tracee.ArgFilter) error {
+	argFilter.Enabled = true
+	// Event argument filter has the following format: "event:argname=argval"
+	// filterName have the format event:argname, and operatorAndValues have the format "=argval"
+	splitFilter := strings.Split(filterName, ":")
+	if len(splitFilter) != 2 {
+		return fmt.Errorf("invalid argument filter format %s%s", filterName, operatorAndValues)
+	}
+	eventName := splitFilter[0]
+	argName := splitFilter[1]
+
+	id, ok := eventsNameToID[eventName]
+	if !ok {
+		return fmt.Errorf("invalid argument filter event name: %s", eventName)
+	}
+
+	eventParams, ok := tracee.EventsIDToParams[id]
+	if !ok {
+		return fmt.Errorf("invalid argument filter event name: %s", eventName)
+	}
+
+	// check if argument name exists for this event
```

Comment:
this should probably be validated in package tracee as well (instead or in addition to this)